### ISSUE TITLE
Somatostatin high in blood

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11510]
**Preferred term label:**
Somatostatin high in blood and a second term
Somatostatinoma on abdominal imaging

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pacak-Zhuang syndrome (https://www.ncbi.nlm.nih.gov/books/NBK621478/)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001466
- label: Elevated circulating somatostatin concentration
- parents: [Abnormal circulating protein concentration (HP:0010876)](https://hpo.jax.org/app/browse/term/HP:0010876)
- pmid: PMID:15496230
- def: The concentration of somatostatin in the blood corculation is above the upper limit of normal.
- comment: Somatostatin is a 14 amino acid polypeptide whose principal function is to regulate growth hormone release from the pituitary. Elevated serum and synovial fluid somatostatin levels have been associated with inflammatory responses most notably in rheumatoid arthritis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11510
- synonyms: Somatostatin high in blood(NONE)

**Comment by MickeySegal** [comment]
Thanks.  I may have been a bit confusing in "Preferred term label" but can we also get an HPO code for Somatostatinoma on abdominal imaging?

**Comment by pnrobinson** [comment]
@MickeySegal 
the HPO would not have separete terms for all of the ways that a finding can be ascertained. I added a short summary of the imaging modalities in the comment though.

Adding new term: 
- ID: HP:6001479
- label: Somatostatinoma
- parents: [Neuroendocrine neoplasm (HP:0100634)](https://hpo.jax.org/app/browse/term/HP:0100634)
- pmid: |PMID:31869077
- def: Somatostatinoma is a rare, functional neuroendocrine tumor (NET) that primarily arises in the pancreas and duodenum, characterized by excessive production of somatostatin.
- comment: A somatostatinoma secretes somatostatin, which typically inhibits numerous gastrointestinal and endocrine functions. Clinically, this condition often presents with a classic triad of diabetes mellitus, cholelithiasis, and steatorrhea, resulting from hormonal suppression. On noncontrast CT, somatostatinomas typically appear isodense relative to pancreatic parenchyma. Endoscopic ultrasound is highly sensitive and specific for detecting pancreatic neuroendocrine neoplasms, including lesions smaller than 5 mm. Somatostatin receptor-based positron emission tomography (PET) using appropriate radiolabeled ligands demonstrates excellent sensitivity and specificity for tumor localization and staging.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11510
- synonyms:
